app/experimenter/nimbus-ui/src/components/Summary/TableBranches/index.test.tsx

```diff
@@ -156,24 +164,24 @@ describe("TableBranches", () => {
               featureValue: null,
               featureEnabled: true,
             },
-            ...MOCK_EXPERIMENT.treatmentBranches!,
           ],
         }}
       />,
     );
 
     const branchTables = screen.queryAllByTestId("table-branch");
-    expect(branchTables).toHaveLength(3);
+    expect(branchTables).toHaveLength(2);
 
     const subjectTable = branchTables[1];
-    for (const name of [
-      "name",
+    for (const property of [
```

Comment:
Took "name" out of this since we shouldn't be showing "not set" for a branch name as it's required for someone to save the branch, and if they haven't saved it yet, it won't show up in the table.